### PR TITLE
Add templatetags

### LINKED ISSUE #9: Open up TemplateTags
We have some template tags that make handling LaTeX files a little easier. We'll need to open them up and test them.

### PR DESCRIPTION
**Pull request by scott-w**
Added some templatetags and tests that make building LaTeX files easier.

Close #9

### CODE REVIEW

File: rest_framework_latex/templatetags/rest_framework_latex.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    value = value.replace('\\', '\\textbackslash{}')
+    value = value.replace('{', '\\{')
+    value = value.replace('}', '\\}')
+    value = value.replace('#', '\\#')
+    value = value.replace('$', '\\$')
+    value = value.replace('%', '\\%')
+    value = value.replace('^', '\\^')
+    value = value.replace('_', '\\_')
+    value = value.replace('~', '\\textasciitidle{}')
+    value = value.replace('\\textbackslash\\{\\}', '\\textbackslash{}')
```
Comment (kennydude) [inline rest_framework_latex/templatetags/rest_framework_latex.py:19]: What's this one doing?

File: rest_framework_latex/templatetags/rest_framework_latex.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def latex_resources():
+    """Return the LATEX_RESOURCES setting with a trailing /
+    """
+    resources = getattr(settings, 'LATEX_RESOURCES', None)
+    if resources is None:
+        raise ImproperlyConfigured(
+            'LATEX_RESOURCES is not defined in settings')
+    if not resources.endswith('/'):
+        resources = u'{}/'.format(resources)
+    return resources
```
Comment (kennydude) [inline rest_framework_latex/templatetags/rest_framework_latex.py]: Not sure what the purpose of this is? We run lualatex in the temporary directory which has a copy of LATEX_RESOURCES?

Comment (scott-w) [inline rest_framework_latex/templatetags/rest_framework_latex.py]: It doesn't seem to use that directory.

File: rest_framework_latex/templatetags/rest_framework_latex.py (hunk 1 above)
Comment (scott-w) [inline rest_framework_latex/templatetags/rest_framework_latex.py:19]: When `\\` is changed to `\\textbackslash{}`, the next two rules change it to `\\textbackslash\\{\\}` - the bottom rule fixes that.

It's less efficient but it gives the correct result.

File: rest_framework_latex/templatetags/rest_framework_latex.py (hunk 2 above)
Comment (scott-w) [inline rest_framework_latex/templatetags/rest_framework_latex.py]: At least when running this on our staging with a different `LATEX_RESOURCES` set

Comment (kennydude) [inline rest_framework_latex/templatetags/rest_framework_latex.py]: It did when I ran it, we set the cwd?

File: rest_framework_latex/templatetags/rest_framework_latex.py (hunk 1 above)
Comment (kennydude) [inline rest_framework_latex/templatetags/rest_framework_latex.py:19]: Ah fair enough

File: rest_framework_latex/templatetags/rest_framework_latex.py (hunk 2 above)
Comment (kennydude) [inline rest_framework_latex/templatetags/rest_framework_latex.py]: We set the working directory when called Lualatex so it should all be fine? https://github.com/mypebble/rest-framework-latex/blob/master/rest_framework_latex/renderers.py#L74

**Pull request merged**